File: CuponSmart/src/cuponsmart/modelo/pojo/respuesta/RespuestaDireccion.java
package cuponsmart.modelo.pojo.respuesta;

import java.util.List;
import cuponsmart.modelo.pojo.entidad.Direccion;

public class RespuestaDireccion extends Mensaje{
    private List<Direccion> contenido;
    
    public List<Direccion> getContenido(){
        return contenido;
    }
    
    public void setContenido(List<Direccion> contenido){
        this.contenido = contenido;
    }
}

File: AppCuponSmart/app/src/main/java/uv/tc/appcuponsmart/data/model/respuesta/RespuestaDireccion.kt
package uv.tc.appcuponsmart.data.model.respuesta

import uv.tc.appcuponsmart.data.model.entidad.Direccion

class RespuestaDireccion(
    error: Boolean,
    mensaje: String,
    var contenido: MutableList<Direccion>? = null
): Mensaje(error, mensaje)